src/views/CardStack/CardStackTransitioner.js

```diff
@@ -62,7 +65,9 @@ class CardStackTransitioner extends React.Component<Props> {
     // props for the old screen
     prevTransitionProps: ?NavigationTransitionProps
   ) => {
+    // const isModal = this._getMode() === 'modal';
```

Comment:
Hey @JulianKingman I'm confused by this line. Should this be used or not? If not we should remove this.

Reply:
I wasn't sure if it was working, but preferred using the getMode() function. It works now.